sdk/resourcemanager/Azure.ResourceManager/src/Common/Custom/Models/ManagedServiceIdentity.Serialization.cs

```diff
@@ -171,11 +213,14 @@ private void AppendChildObject(StringBuilder stringBuilder, object childObject,
         internal static ManagedServiceIdentity DeserializeManagedServiceIdentity(JsonElement element, ModelReaderWriterOptions options, JsonSerializerOptions jOptions)
```

Comment:
similar comment applies to this method as well: this `jOptions` after our modifications, is never used, we could remove it and we could simplify our overloads.

Reply:
we should just remove the duplicated Write and DeserializeXXX methods, move the implementation to the actual internal methods.